bitnami/moodle/values.yaml

```diff
@@ -523,7 +523,7 @@ metrics:
 # Add custom certificates and certificate authorities to redmine container
 certificates:
   customCertificate:
-    certificateSecret: ""
+    certificateSecret:
```

Comment:
Please don't change the default values.

Reply:
this was an accident when I was doing some testing.